### ISSUE TITLE
Schema converter support

### ISSUE THREAD

**Issue by Yuutakasan** [Apicurio/apicurio-registry#2675]
It is time-consuming because we have to deal with json-schema for open api spec, Avro for Kafka, and another schema for Prptbuf. 
I wonder if it would be possible to add a function to be able to convert each schema information.

**Comment by EricWittmann** [comment]
This is a very interesting idea.  I suspect converting from one schema format to another is a lossy process.  Certainly the semantic domain of JSON Schema is much greater than Avro (for example).

So I think there are a lot of gotchas and challenges with this.  But if it's something that we could get to work, even in a limited/constrained way, it could be very valuable.

@Apicurio/developers wdyt?

**Comment by Yuutakasan** [comment]
Yes, it is. I am currently using apicurio studio to create openapi specs and define API responses in json, but the backend is kafka and I am having trouble with having to redefine the interaction with kafka in avro.

**Comment by EricWittmann** [comment]
Perhaps a better approach would be to use AsyncAPI (instead of OpenAPI) to describe your Kafka applications?

https://www.asyncapi.com/

Apicurio Studio has **beta** support for AsyncAPI.

**Comment by Yuutakasan** [comment]
I will try it once.

**Comment by Yuutakasan** [comment]
Finally tried it...GUI is working properly now. I hadn't noticed. It's true that this feature might be a good substitute for now.
